R/model.R

```diff
@@ -761,6 +761,9 @@ CmdStanModel$set("public", name = "check_syntax", value = check_syntax_method)
 #'   quantities block. If the parameters block is empty then using
 #'   `fixed_param=TRUE` is mandatory. When `fixed_param=TRUE` the `chains` and
 #'   `parallel_chains` arguments will be set to `1`.
+#'   * `chain_id`: (vector) A vector of chain IDs. Must contain `chains` unique
+#'   positive integers. If not set, the default chain IDs are used (integers
+#'   starting from `1`).
```

Comment:
Should we document this higher up next to the other arguments related to the chains? On the other hand, this argument won't be used much so I can also see a reason to keep it documented here at the bottom.

Reply:
Moved up. I think that makes sense.